certbot-apache/certbot_apache/tests/centos6_test.py

```diff
@@ -0,0 +1,125 @@
+"""Test for certbot_apache.configurator for CentOS 6 overrides"""
+import os
+import unittest
+
+from certbot_apache import obj
+from certbot_apache import override_centos
+from certbot_apache import parser
+from certbot_apache.tests import util
+
+def get_vh_truth(temp_dir, config_name):
+    """Return the ground truth for the specified directory."""
+    prefix = os.path.join(
+        temp_dir, config_name, "httpd/conf.d")
+
+    aug_pre = "/files" + prefix
+    vh_truth = [
+        obj.VirtualHost(
+            os.path.join(prefix, "test.example.com.conf"),
+            os.path.join(aug_pre, "test.example.com.conf/VirtualHost"),
+            set([obj.Addr.fromstring("*:80")]),
+            False, True, "test.example.com"),
+        obj.VirtualHost(
+            os.path.join(prefix, "ssl.conf"),
+            os.path.join(aug_pre, "ssl.conf/VirtualHost"),
+            set([obj.Addr.fromstring("_default_:443")]),
+            True, True, None)
+    ]
+    return vh_truth
+
+class CentOS6Tests(util.ApacheTest):
+    """Tests for CentOS 6"""
+
+    _multiprocess_can_split_ = True
```

Comment:
Now that we've switched to pytest from nose, we don't need to set values like this.